core/src/main/java/org/kiwix/kiwixmobile/core/main/ToolbarScrollingKiwixWebView.java

```diff
@@ -51,6 +51,17 @@ public ToolbarScrollingKiwixWebView(Context context, WebViewCallback callback, A
     this.toolbarView = toolbarView;
     this.bottomBarView = bottomBarView;
     this.sharedPreferenceUtil = sharedPreferenceUtil;
+    if (toolbarView != null) {
+      fixInitalScrollingIssue();
+    }
+  }
+
+  /**
+   * The webview needs to be scrolled with 0 to not be slightly hidden on startup.
+   * See https://github.com/kiwix/kiwix-android/issues/2304 for issue description.
+   */
+  private void fixInitalScrollingIssue() {
+    moveToolbar(0);
```

Comment:
Have a look here, this wasn't implemented in custom in one-activity

Reply:
Removing this, considering that it'll be considered while merging this in develop in future and for making a clean _base-off_ from _feature/one-activity_